tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs

```diff
@@ -253,11 +264,11 @@ void UpsertVstsReleaseDeploymentToDb(SqlConnection sqlConnection, int releaseEnv
             cmd.Parameters.Add(new SqlParameter("@ReleaseEnvironmentId", releaseEnvironmentId));
             cmd.Parameters.Add(new SqlParameter("@Attempt", deployment.Attempt));
             cmd.Parameters.Add(new SqlParameter("@Status", deployment.Status.ToString()));
-            cmd.Parameters.Add(new SqlParameter("@LastModifiedOn", deployment.LastModifiedOn));
+            cmd.Parameters.Add(new SqlParameter("@LastModifiedOn", SqlDbType.DateTime2) { Value = deployment.LastModifiedOn });
```

Comment:
Any reason why you needed to change the type?

Reply:
Previously the type is wrong, it is the right type to use; check what is done for vsts build.